javax/src/main/java/org/jboss/ejb/protocol/remote/Protocol.java

```diff
@@ -23,7 +23,11 @@
  */
 final class Protocol {
 
-    public static final int LATEST_VERSION = 3;
+    static final int JAVAEE_PROTOCOL_VERSION = 3;
+    static final int JAKARTAEE_PROTOCOL_VERSION = 4;
```

Comment:
I really would like to validate if this is necessary or if a simple attribute in the remote outbound connection could be used to indicate the need for transformation without an upgrade in the protocol version